content/1-triggering-jenkins/guide.md

```diff
@@ -0,0 +1,122 @@
+# 1. Triggering a Jenkins build from a push to GitHub
+
+Guidance on integrating Kobiton service into the build pipeline of mobile app development: Jenkins, HockeyApp, Appium and ReactNative
+
+**Triggering a Jenkins build from a push to GitHub**
```

Comment:
The current content has following problems:

- distractions on information (e.g. lilyhdo/pipeline-test, docker)
- the reading flow is not clear (new line, paragraph)

Here is my suggest structure

```
## Triggering a Jenkins build from a push to Github

#### 1.1 Install Github Integration plugin on Jenkins

....

#### 1.2 Configure Github repo to push to Jenkins

...

#### 1.3 Allow Jenkins to access to the Github repository

...

```

(use `1.1` to consistent with other section)

On README say something

```
Let's assume we're a company having a mobile app, and has a domain https://awesome-mapp.com. We're developing the app and we need to integrate Kobiton into our development and testing pipeline. At awesome-mapp, we uses

- Jenkins for build the app
- HocketApps for keeping the daily app build
...
```

Notes for improvement:

- Make sure the content organization looks clear
- No need to mention docker, just assume user needs to have a Jenkins server available with public domain , say `ci.awesome-mapp.com`
- Avoid using external link like https://www.thegeekstuff.com/2016/10/jenkins-git-setup/
- Skip `Add GitHub project to Jenkins configuration`